spec/controllers/api/v1/movies_controller_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'rails_helper'
+
+module Api::V1
+  # Movies::MoviesController controller
+  RSpec.describe Movies::MoviesController, type: :controller do
+    describe 'POST movies#create' do
+
+      context 'when create a movie' do
+
+        include_context 'Authenticated User'
+        let(:person){ create(:person) }
+        let(:movie){ create(:movie) }
+        let(:movie_params) do
+          {
+            'movie': {
+              'title': Faker::DcComics.title,
+              'release': Faker::Date.in_date_period,
+              'person_ids': Person.all.shuffle[1..5].map(&:id)
+            }
+          }
+        end
+
+        before do
+          post :create, params: movie_params
+        end
+
+        it 'responds with status created' do
+          expect(response).to have_http_status(:created)
+        end
+
+        it 'responds with the object created' do
```

Comment:
add wrong cases to tests, what is the expected when a movie is not found or when a movie is not created just fine